pyproject.toml

```diff
@@ -71,6 +71,14 @@ lint.extend-select = [
 lint.isort = {known-first-party = ["helpers", "package_info", "pipx"]}
 lint.mccabe.max-complexity = 15
 
+[tool.codespell]
+# Ref: https://github.com/codespell-project/codespell#using-a-config-file
+skip = '.git,*.pdf,*.svg,.nox'
+check-hidden = true
+# case sensitive etc
+ignore-regex = '\b(UE|path/doesnt/exist)\b'
+ignore-words-list = 'astroid'
```

Comment:
I guess we don't need this, yet?

```suggestion
```

Reply:
```
❯ codespell
./testdata/tests_packages/macos21-python3.12.txt:18: astroid ==> asteroid
./testdata/tests_packages/macos21-python3.12.txt:19: astroid ==> asteroid
./testdata/tests_packages/unix-python3.10.txt:18: astroid ==> asteroid
./testdata/tests_packages/unix-python3.10.txt:19: astroid ==> asteroid
./testdata/tests_packages/unix-python3.11.txt:18: astroid ==> asteroid
./testdata/tests_packages/unix-python3.11.txt:19: astroid ==> asteroid
./testdata/tests_packages/unix-python3.12.txt:18: astroid ==> asteroid
./testdata/tests_packages/unix-python3.12.txt:19: astroid ==> asteroid
./testdata/tests_packages/unix-python3.8.txt:18: astroid ==> asteroid
./testdata/tests_packages/unix-python3.8.txt:19: astroid ==> asteroid
./testdata/tests_packages/unix-python3.9.txt:18: astroid ==> asteroid
./testdata/tests_packages/unix-python3.9.txt:19: astroid ==> asteroid
./testdata/tests_packages/win-python3.12.txt:18: astroid ==> asteroid
./testdata/tests_packages/win-python3.12.txt:19: astroid ==> asteroid
```

sure thing we could just ignore the `testdata`. What would you prefer?

Reply:
Let's do the latter. I've edited the suggestion accordingly.

Reply:
thanks! just note that `skip` is already set above to some other globs -- I think this latter would override it. Please suggest on that line , or just let me know - I can do that

Reply:
done